packages/abstract-core/src/native/version_control.rs

```diff
@@ -40,6 +44,15 @@ pub mod state {
     pub const REGISTERED_MODULES: Map<&ModuleInfo, ModuleReference> = Map::new("module_lib");
     // Yanked Modules
     pub const YANKED_MODULES: Map<&ModuleInfo, ModuleReference> = Map::new("yanked_modules");
+    // Modules Fee
+    pub const MODULE_MONETIZATION: Map<(Namespace, String), Monetization> =
+        Map::new("yanked_modules");
+
+    pub fn load_module_monetization(deps: Deps, key: (Namespace, String)) -> Monetization {
```

Comment:
these kinds of helpers are dangerous to expose, a user might think they can use this in their contract while it can actually only be called in the version control contract. Better to remove / move it.

Reply:
Removed here : https://github.com/AbstractSDK/contracts/pull/358/commits/6f5bf715249fc44e62474650c59cb78fd8def35c#diff-4e6d021cf259966a3fb05c3c5e0a06618d9c2962705c9e38c425d3885ea85ad2L51